### ISSUE TITLE
Propose `get_detailed_change_log`

### ISSUE THREAD

**Issue by bdpedigo** [seung-lab/CAVEclient#148]
there are currently functions for `get_operation_details` https://caveclient.readthedocs.io/en/latest/api/caveclient.html#caveclient.chunkedgraph.ChunkedGraphClientV1.get_operation_details 

and `get_tabular_change_log` https://caveclient.readthedocs.io/en/latest/api/caveclient.html#caveclient.chunkedgraph.ChunkedGraphClientV1.get_tabular_change_log

which have mostly distinct (but some overlapping) properties

I propose a `get_detailed_change_log` function which would combine the two into a unified table. Example of what this could look like here https://github.com/bdpedigo/skedits/blob/be0aa5f12ca8a319fb7ab4f8e3d8a355daa2c165/pkg/pkg/edits/changes.py#L38

Basically the naive version of this would just do a join of the two functions above, all client-side

Would a PR for this be accepted? Or other ideas for consolidating the two?
